src/core/src/op/divide.cpp

```diff
@@ -96,109 +82,108 @@ bool evaluate_bound(const Node* node, ov::TensorVector& output_values, bool is_u
+    auto less_up_outputs = TensorVector{{element::boolean, input2.get_shape()}};
     auto& input2_positive_up_mask = less_up_outputs.front();
 
-    bool status = op::v1::Less().evaluate(less_up_outputs, ov::TensorVector{zero_t, input2_up});
+    bool status = Less().evaluate(less_up_outputs, TensorVector{zero_t, input2_up});
     if (!status)
         return status;
 
     // mask to find out negative values for arg2
-    auto less_low_outputs = ov::TensorVector{{element::boolean, input2.get_shape()}};
+    auto less_low_outputs = TensorVector{{element::boolean, input2.get_shape()}};
     auto& input2_negative_low_mask = less_low_outputs.front();
-    status = op::v1::Less().evaluate(less_low_outputs, {input2_low, zero_t});
+    status = Less().evaluate(less_low_outputs, {input2_low, zero_t});
     if (!status)
         return status;
 
     // mask to find out ranges around 0 for arg2
-    auto logical_and_up_outputs = ov::TensorVector{{element::boolean, input2.get_shape()}};
+    auto logical_and_up_outputs = TensorVector{{element::boolean, input2.get_shape()}};
     auto& input2_low_negative_up_positive_mask = logical_and_up_outputs.front();
-    status = op::v1::LogicalAnd().evaluate(logical_and_up_outputs, {input2_negative_low_mask, input2_positive_up_mask});
+    status = LogicalAnd().evaluate(logical_and_up_outputs, {input2_negative_low_mask, input2_positive_up_mask});
     if (!status)
         return status;
 
-    auto value1_outs = ov::TensorVector{{input1.get_element_type(), input_shape.get_shape()}};
+    auto value1_outs = TensorVector{{input1.get_element_type(), input_shape.get_shape()}};
     auto& value1 = value1_outs.front();
 
-    auto value2_outs = ov::TensorVector{{input2.get_element_type(), input2.get_shape()}};
+    auto value2_outs = TensorVector{{input2.get_element_type(), input2.get_shape()}};
     auto& value2 = value2_outs.front();
 
     if (!is_upper) {
-        status = op::v1::Select().evaluate(value1_outs, {input2_positive_up_mask, input1_low, input1_up});
+        status = Select().evaluate(value1_outs, {input2_positive_up_mask, input1_low, input1_up});
         if (!status)
             return status;
 
-        status = op::v1::Select().evaluate(value2_outs, {input2_positive_up_mask, input2_up, input2_low});
+        status = Select().evaluate(value2_outs, {input2_positive_up_mask, input2_up, input2_low});
         if (!status)
             return status;
 
-        status = node->evaluate(output_values, ov::TensorVector{value1, value2});
+        status = node->evaluate(output_values, TensorVector{value1, value2});
         if (!status)
             return status;
 
         // replace values where zeros inside range of second arg to maximum values
         OPENVINO_SUPPRESS_DEPRECATED_START
-        auto output_minimum_value = get_constant_min_of_type(output_values[0].get_element_type());
+        const auto output_minimum_value = ngraph::get_constant_min_of_type(output_values[0].get_element_type());
```

Comment:
Consider migrate this function based on `ov::util::make_tensor_of_max_value` in validation util, then `ngraph` can be removed completely from this operator.

Will also remove creation of `out_min_v ` tensor from const.

Reply:
`ov::util::make_tensor_of_min_value` created and used